e2e/pages/caseView/caseView.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     ordersHearing: 'Orders and hearing',
     casePeople: 'People in the case',
     legalBasis: 'Legal basis',
-    documents: 'Documents',
+    documents: 'cut-tabs>div>ul>li>a[href*=Documents]',
+    attachedPdf: 'ccd-read-document-field>a',
```

Comment:
Selector does not really match the name because selector is generic enough to select any document link. I suggest renaming it to something more generic e.g. 'documentLink'.

Please also move it to `e2e/paths/caseSubmitted_test.js` as this is not best place because `e2e/pages/caseView/caseView.js` has generic. elements shared across all the tabs.

Reply:
Updated named `attachedPdf` to `documentLink` and also `documentLink` moved to `caseSubmitted_test.js`